simpl/config.py

```diff
@@ -526,7 +525,7 @@ def get_defaults(self):
         parsed, _ = parser.parse_known_args([])
         return vars(parsed)
 
-    def parse_ini(self, paths=None, namespace=None):
+    def parse_ini(self, paths=None, namespace=None, permissive=False):
```

Comment:
Supporting different parsing between ini files and command arguments will make the interface more complicated than needed. So I think if we add it we should add `permissive=` kwarg (instead of `permissive_ini=, permissive_args, etc..`
But... having said that... we can leave this out until it is required. As you say, we don't need it for our use cases, so we could skip it until (if ever) someone asks for it.
In other words, the code as is is good, IMO